frontend/src/admin/pages/[groupId]/contest/[id]/index.vue

```diff
@@ -16,6 +17,7 @@ interface Row {
 }
 
 const toggle = ref(false)
+const toggleNotice = ref(false)
```

Comment:
변수 이름이 헷갈릴 것 같아서 `showImportModal`, `showNoticeModal` 이런 식으로 통일해주면 좋을 것 같아요!

```suggestion
const showImportModal = ref(false)
const showNoticeModal = ref(false)
```